src/Microsoft.ML.TimeSeries/TimeSeriesStatic.cs

```diff
@@ -0,0 +1,269 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.TimeSeriesProcessing;
+using Microsoft.ML.StaticPipe.Runtime;
+using System.Collections.Generic;
+using static Microsoft.ML.Runtime.TimeSeriesProcessing.SequentialAnomalyDetectionTransformBase<System.Single, Microsoft.ML.Runtime.TimeSeriesProcessing.IidAnomalyDetectionBase.State>;
+using static Microsoft.ML.Runtime.TimeSeriesProcessing.SequentialAnomalyDetectionTransformBase<System.Single, Microsoft.ML.Runtime.TimeSeriesProcessing.SsaAnomalyDetectionBase.State>;
+
+namespace Microsoft.ML.StaticPipe
+{
+    /// <summary>
+    /// IidChangePoint static API extension methods.
+    /// </summary>
+    public static class IidChangePointStaticExtensions
+    {
+        private sealed class OutColumn : Vector<double>
+        {
+            public PipelineColumn Input { get; }
+
+            public OutColumn(
+                Scalar<float> input,
+                int confidence,
+                int changeHistoryLength,
+                SequentialAnomalyDetectionTransformBase<System.Single, Microsoft.ML.Runtime.TimeSeriesProcessing.IidAnomalyDetectionBase.State>.MartingaleType martingale,
```

Comment:
>System.Single, Microsoft.ML.Runtime.TimeSeriesProcessing.IidAnomalyDetectionBase.State [](start = 56, length = 86)

A) please don't use System.Single, it's just a float
B) you can give `Microsoft.ML.Runtime.TimeSeriesProcessing.IidAnomalyDetectionBase.State` pretty name in using and reuse it everywhere.

Reply:
Thanks!

---
In reply to: [236426516](https://github.com/dotnet/machinelearning/pull/1656#discussion_r236426516) [](ancestors = 236426516)